docs/imviz/import_data.rst

```diff
@@ -99,7 +99,7 @@ object, you can load it into Imviz as follows:
     from jdaviz import Imviz
 
     # mydatamodel is a jwst.datamodels object
-    ndd = NDData(np.array(mydatamodel.data), wcs=mydatamodel.get_fits_wcs())
+    ndd = NDData(mydatamodel.data, StdDevUncertainty(mydatamodel.err), mydatamodel.dq, mydatamodel.meta.wcs, meta=dict(mydatamodel.meta.items()))
```

Comment:
I am not familiar with all flavors of JWST data. Is it guaranteed to always provide std dev uncertainty (as opposed to IVAR or VAR)?